pkg/services/sqlstore/migrations/object_store_mig.go

```diff
@@ -124,19 +134,16 @@ func addObjectStorageMigrations(mg *migrator.Migrator) {
 	// Migration cleanups: given that this is a complex setup
 	// that requires a lot of testing before we are ready to push out of dev
 	// this script lets us easy wipe previous changes and initialize clean tables
-	suffix := " (v0)" // change this when we want to wipe and reset the object tables
+	suffix := " (v2)" // change this when we want to wipe and reset the object tables
 	mg.AddMigration("ObjectStore init: cleanup"+suffix, migrator.NewRawSQLMigration(strings.TrimSpace(`
 		DELETE FROM migration_log WHERE migration_id LIKE 'ObjectStore init%';
-		DROP table if exists "object"; 
```

Comment:
mysql was not happy with the multiple commands in one shot... so this adds an explicit drop before each add table